site/_resources/upgrading.md

```diff
@@ -5,126 +5,122 @@ layout: page
-You'll need to re-check where your DNS names are pointing as well, using [Get your hostname or IP address]({% link _guides/deploy-options.md %}#get_your_hostname_or_ip_address).
-
-## The less easy way
-
-This section contains information for administrators who wish to apply the Contour 1.0.0-beta.1 to Contour 1.0.0-rc.1 changes manually.
-
-### Upgrade to Contour 1.0.0-rc.1
+### Update deprecated `contour.heptio.com` annotations
 
-Change the Contour image version to `docker.io/projectcontour/contour:v1.0.0-rc.1`.
+All the annotations with the prefix `contour.heptio.com` have been
+migrated to their respective `projectcontour.io` counterparts.  The
+deprecated `contour.heptio.com` annotations will be recognized
+through the Contour 1.0 release, but are scheduled to be removed after
+Contour 1.0.
 
-### Recommended Envoy version
+See the [annotation documentation]({% link _docs_1_0/annotations.md %})
+for more information.
 
-The recommended version of Envoy remains unchanged.
-Ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.11.2`.
+### Update old `projectcontour.io/v1alpha1` group versions
 
-### projectcontour.io/v1 group version
+If you are upgrading a cluster that you previously installed a 1.0.0
+release candidate, note that Contour 1.0.0 moves the HTTPProxy CRD
+from `projectcontour.io/v1alpha1` to `projectcontour.io/v1`.  and
+will no longer recognize the former group version.
 
-Contour 1.0.0-rc.1 moves the HTTPProxy CRD from `projectcontour.io/v1alpha1` to `projectcontour.io/v1`.
-Contour will no longer recognize the former group version.
-Please edit your HTTPProxy documents to update their group verision to `projectcontour.io/v1`.
+Please edit your HTTPProxy documents to update their group verision
+to `projectcontour.io/v1`.
 
-### HTTPProxy v1 schema changes
+### Check for HTTPProxy v1 schema changes
 
-As part of finalizing the HTTPProxy v1 schema, three breaking changes have been introduced.
+As part of finalizing the HTTPProxy v1 schema, three breaking changes
+have been introduced.  If you are upgrading a cluster that you
+previously installed a Contour 1.0.0 release candidate, you may
+need to edit HTTPProxy object to conform to the upgraded schema.
 
-#### Per service heath checking has moved to per route
+* The per-route prefix rewrite key, `prefixRewrite` has been removed.
+  See [#899](https://github.com/projectcontour/contour/issues/899)
+  for the status of its replacement.
 
-The per service health check key, `healthcheck` has moved to per route and has been renamed `healthCheckPolicy`.
+* The per-service health check key, `healthcheck` has moved to
+  per-route and has been renamed `healthCheckPolicy`.
 
-Before:
+<table class="table table-borderless" style="border: none;">
+<tr><th>Before:</th><th>After:</th></tr>
 
-```yaml
+<tr>
+<td><pre><code class="language-yaml" data-lang="yaml">
```

Comment:
is this a copy and paste from some other text, why did you move away from the

before

after

style?

I don't mind it, but this feels like something we can do later across the whole site.

Reply:
I found it hard to compare the before and after, so I wanted to put them next to one another. You can't do that in a Markdown table, so one think led to another ...

Reply:
I don't like the direct HTML inclusion, but those differences are *much* easier to see. I think we should do this one, but try and find a better table solution for the rest of the documentation.

Reply:
Thanks for wrestling with this, let’s keep working on harmonising the formatting next week.